lib/services/patcher_api.dart

```diff
@@ -268,9 +271,9 @@ void sharePatchedFile(String appName, String version) {
   }
 }
 
-String _getFileName(String appName, String version) {
+String _getFileName(String appName, String version, {String patchVersion=''}) {
   final String prefix = appName.toLowerCase().replaceAll(' ', '-');
-  final String newName = '$prefix-revanced_v$version.apk';
+  final String newName = '$prefix-revanced_v$version$patchVersion.apk';
```

Comment:
Maybe `some.app.package-v1.2.3-revanced-v3.0.0.apk`:

```suggestion
  final String newName = '$prefix-v$version-revanced-v$patchVersion.apk';
```